.github/pull_request_template.md

```diff
@@ -1,21 +1,38 @@
+<!-- Use the first line of the commit description as the title for the PR -->
+
 ## Description
 
-Describe the content of the PR and links to related issues, bugs of features.
+<!--
+Delete this comment and add a proper description of the changes contained in this PR. The text here will be used in the commit message since the approved PRs are always squash-merged. The preferred format is:
 
-## Requirements
+---
+<type>[<scope>]: <one-line-summary>
+
+[description of the main changes with links to appropriate documents/issues/references/...]
 
-Before submitting this PR, please make sure:
+Reviewed-by: John Who <github_handle | email>
```

Comment:
... except that. I think it's enough that the commit contains the #pr.

Reply:
Deleted.